remote/index.js

```diff
@@ -66,7 +66,8 @@ module.exports = function (app) {
       vid: app._backstop.testCtr
     };
 
-    backstop('test', { config }).then(
+    const command = req.path.includes('dtest') ? 'test' : 'reference';
```

Comment:
I wonder if we should put leading and trailing slashes around `/dtest/` to like, make sure data in `:testId` and `:scenarioId` don't accidentally trip the test?